build/settings.js

```diff
@@ -282,6 +282,11 @@ function execute(config) {
     settings['devtools.qps.enabled'] = true;
   }
 
+  if (config.DOGFOOD === '1') {
+    settings['debug.performance_data.spark_dogfooding'] = '1';
```

Comment:
I think we should generally avoid conflating dogfooding and Spark. They support each other, but they aren't the same thing. Thus I would suggest a name 'debug.performance_data.dogfooding' instead.